bucket/avro-tools.json

```diff
@@ -0,0 +1,25 @@
+{
+    "version": "1.8.2",
+    "description": "A collection of Avro tools. Data serialization system.",
+    "homepage": "http://avro.apache.org/",
+    "license": "Apache-2.0",
+    "url": "http://apache.org/dist/avro/avro-1.8.2/java/avro-tools-1.8.2.jar#/avro-tools.jar",
+    "hash": "sha1:80e4519289ef9c5a09280639c8c9a669d0a4704d",
+    "bin": "avro-tools.jar",
+    "suggest": {
+        "JDK": [
+            "java/oraclejdk",
+            "java/openjdk"
+        ]
+    },
+    "checkver": {
+        "url": "http://avro.apache.org/releases.html",
+        "regex": "Avro ([\\d.]+) Released"
+    },
+    "autoupdate": {
+        "url": "http://apache.org/dist/avro/avro-$version/java/avro-tools-$version.jar#/avro-tools.jar",
```

Comment:
```suggestion
        "url": "https://www.apache.org/dist/avro/avro-$version/java/avro-tools-$version.jar#/avro-tools.jar",
```

Reply:
I used `http` because main page is not https so i did not change url. But if the file is available then yes, `https` would be better